pkg/apis/certmanager/v1/types.go

```diff
@@ -203,7 +203,15 @@ const (
 	UsageSigning KeyUsage = "signing"
 	// UsageDigitalSignature is for digital signature
 	UsageDigitalSignature KeyUsage = "digital signature"
-	// UsageContentCommittment is content commitment
+	// UsageContentCommitment is content commitment
+	UsageContentCommitment KeyUsage = "content commitment"
+
+	// UsageContentCommittment is deprecated
+	//
+	// This misspelling will live forever in our hearts and will remind us
+	// of the great Unix dictionary that was not able to spell "referrer" in 1992.
+	//
+	// Deprecated: use UsageContentCommitment
 	UsageContentCommittment KeyUsage = "content commitment"
```

Comment:
It appears these two are the same string on the other end - can we instead just replace the old one with the new? The actual value hasn't changed, so this only breaks our Golang API consumers rather than our K8s API consumers

Reply:
I hope you aren't asking me.

I don't claim to understand deprecation, go deprecation, or cert-manager deprecation.

I'd certainly prefer to entirely remove a misspelled word as opposed to include a note reminding people of it.

Reply:
I was asking you in-so-much as you are the pr author, so you're the one with the power to make the change 😄

Reply:
Totally happy to do it if you're ok w/ it.

I really don't understand what the api commitments are for these things. Hence my default of being pessimistic.